src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,18 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: complete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
The synchronisation pipeline (you did this for subclass of and deprecation) deliberatly picks and chooses content that we want synchronised. I would prefer you to add a robot remove step that gets rid of all of these properties as part of the ingest to keep the component file smaller.

Reply:
Synchronisation, OK thought so. So when we integrate a source, we do a mapping process. And then there's synchronisation. I just want to give myself clarity; these two processes cover like 100% of our source ingestion, you would say?

- [ ] `robot remove` these 4 properties @joeflack4

Reply:
- [ ] Advise @matentzn @twhetzel 
In addition to just clarifying the above question, I want to know if it really matters to remove `releaseId` and ` releaseDate`. Because they only exist on the `Ontology` itself, not any class. And the command I ran to remove them didn't work, so I undid it.

<details><summary>Details</summary>
<p>

I ran: `remove --term http://id.who.int/icd/schema/narrowerTerm --term http://id.who.int/icd/schema/browserUrl --term http://id.who.int/icd/schema/codingHint --term http://id.who.int/icd/schema/releaseDate --term http://id.who.int/icd/schema/releaseId --select "annotation-properties"`

For `narrowerTerm`,  `browserUrl`, and `codingHint`, it removed usages and declarations as I intended. But for `releaseId` and `releaseDate`, it only removed the declarations.

`cat tmp/component-download-icd11foundation.owl.owl | grep releaseDate`
```owl
        <schema:releaseDate>2023-04-09</schema:releaseDate>
    <!-- http://id.who.int/icd/schema/releaseDate -->
    <owl:AnnotationProperty rdf:about="http://id.who.int/icd/schema/releaseDate"/>
```

`cat tmp/component-download-icd11foundation.owl.owl | grep releaseId`
```owl
        <schema:releaseId xml:lang="en">beta</schema:releaseId>
    <!-- http://id.who.int/icd/schema/releaseId -->
    <owl:AnnotationProperty rdf:about="http://id.who.int/icd/schema/releaseId"/>
```

As far as I see it, my choices are:
a. Actually just leave these in, they're harmless and maybe even nice to keep.
b. Stay with [robot remove](https://robot.obolibrary.org/remove#selectors)'ing them, but how? I can try `--select "properties"` instead of `--select "annotation-properties"`.

</p>
</details>